### ISSUE TITLE
No UUID

### ISSUE THREAD

**Issue by amaccuish** [digint/btrbk#440]
So reading the FAQ, it seems I am hit by "Aborted: subvolume has no UUID", in that my subvolume 5 does not have a UUID. How can I go about setting one?

> /
        Name:                   <FS_TREE>
        UUID:                   86334f48-757e-4465-b8f3-5c2714764b9d
        Parent UUID:            -
        Received UUID:          -
        Creation time:          2020-04-30 20:19:14 +0100
        Subvolume ID:           5
        Generation:             44952077
        Gen at creation:        0
        Parent ID:              0
        Top level ID:           0
        Flags:                  -
        Snapshot(s):

**Comment by amaccuish** [comment]
I can see talk of it here but not sure it went anywhere..: https://www.spinics.net/lists/linux-btrfs/msg76016.html

**Comment by jwatt** [comment]
Your only hope of doing that would be to use [btrfstune](https://btrfs.readthedocs.io/en/latest/btrfstune.html). No idea if it will work (or malfunction) when used with a filesystem that's in that state though.

Also, this isn't a btrbk issue.

**Comment by amaccuish** [comment]
Yes, not a btrbk issue but had hoped someone might have a solution. No answer on the btrfs mailing list https://lore.kernel.org/linux-btrfs/02c7933d-e444-0eba-e46f-2cab8e7601fa@maccuish.de/

Do you know anywhere else I could get help? I tried btrfstune but it changes the FS UUID, not the subvolume UUID.

**Comment by frinnst** [comment]
I found this thread when my backups stopped running. In my case it's related to btrfsprogs 6.1. Downgrading to 6.0 allows the backup to run. Obviously not the cause of the original issue since 6.1 was released in December 2022 - just thought i'd leave a comment.

**Comment by digint** [comment]
@frinnst you have hit a bug in btrfs-progs, see #519
